src/components/shared/MarkerSettings.css

```diff
@@ -14,20 +14,10 @@
   flex: 1;
 }
 
-.markerSettingsLabel {
-  display: inline-flex;
-  height: 25px;
-  flex-flow: row nowrap;
-  align-items: center;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  user-select: none;
-}
-
-.markerSettingsSearchbar {
-  padding: 0 5px;
+.markerSettingsSearchField .panelSearchFieldInput {
+  width: 250px;
```

Comment:
Can this be done better? To shorten the width for the marker settings search field, it reaches into class names that really should be implementation details for the `PanelSearch` component.

Reply:
Hmm... I'm not sure my favorite pattern for this. I've done it where you can inject a custom class name, or assign that value programmatically with a CSS variable. I think this is fine though. It's a little "ugly" in that we have some nesting, but it's a simple fix.